### PR TITLE
Fix `InteractingScp330EventArgs::Candy`

### PR DESCRIPTION
**Pull request by VALERA771**
## Description
**Describe the changes** 
Fixing `InteractingScp330EventArgs::Candy`

**What is the current behavior?** (You can also link to an open issue here)
Candy is incorrect
#118

**What is the new behavior?** (if this is a feature change)
Candy is correct

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

**Other information**:

<br />

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [x] I have checked the project can be compiled
- [x] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [x] I have checked no IL patching errors in the console

### Other
- [ ] Still requires more testing

### CODE REVIEW

File: EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs [hunk 1]
```diff
@@ -31,12 +31,23 @@ public class InteractingScp330EventArgs : IPlayerEvent, IScp330Event, IDeniableE
...
-            Candy = Scp330Candies.GetRandom();
             UsageCount = usage;
             ShouldSever = usage >= 2;
             ShouldPlaySound = true;
             IsAllowed = Player.IsHuman;
+
+            if (Scp330Bag.TryGetBag(player.ReferenceHub, out Scp330Bag scp330Bag))
+            {
+                Scp330 = (Scp330)Item.Get(scp330Bag);
+            }
```
Comment (iamalexrouse) [inline EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs:42]:
```suggestion
                Scp330 = (Scp330)Item.Get(scp330Bag);

```

**Review by iamalexrouse — APPROVED** [review]
Micro styling fix. Otherwise, looks good.

File: EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs (hunk 1 above)
Comment (VALERA771) [reply EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs:42]: Stylecop doesn't like this

Comment (iamalexrouse) [reply EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs:42]: > Stylecop doesn't like this

And I don't like stylecop.

Comment (VALERA771) [reply EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs:42]: ☠️☠️☠️💀

**Review by louis1706 — APPROVED** [review]

File: EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs (hunk 1 above)
Comment (louis1706) [reply EXILED/Exiled.Events/EventArgs/Scp330/InteractingScp330EventArgs.cs:42]: > > Stylecop doesn't like this
> 
> And I don't like stylecop.

He can't be cause he have multiple line in the else

**Pull request merged**